release_management/create_release.py

```diff
@@ -425,6 +429,13 @@ def _get_parser():
         help="stop before pushing any changes to GitHub repo",
     )
 
+    parser.add_argument(
+        "--git-repo",
+        default=None,
```

Comment:
Actually, you don't need to bother to specify the default keyword argument, for the default value for that argument is ```None```.

Reply:
Resolved